extensions/amp-analytics/0.1/test/test-linker-manager.js

```diff
@@ -756,7 +773,7 @@ describes.realWin('Linker Manager', {amp: true}, env => {
       return linkerManager.init().then(() => {
         expect(beforeSubmitStub.calledOnce).to.be.true;
         toggleExperiment(win, 'linker-form', false);
-        return expect(beforeSubmitStub).calledWith(sinon.match.func);
```

Comment:
I thought these would silently fail without return? I could be wrong.

Reply:
do you mean you had to use `return`? return is only necessary if the following statement is a promise.

Reply:
I just double-checked and you are right :)